tests/modules/accordion.spec.tsx

```diff
@@ -0,0 +1,122 @@
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import { Accordion, AccordionProps, Text } from "../../src";
+import { DEFAULT_TEXT, DEFAULT_TITLE } from "../common";
+
+const renderComponent = (
```

Comment:
these functions can be moved to the end of the tests with a H1 divider indicating `TEST HELPERS`